### PR TITLE
LTP:Fix for not closing file descriptors and a formatting issue - setreuid07

### PR DESCRIPTION
**Pull request by ashokchandran**
issue: was not closing fds opened in a thread and a formatting issue observed.
fix: close the fds after use and fix formatting

### CODE REVIEW

**Comment by letmaik** [comment]
Can you prefix all your LTP patch PRs with "LTP: " or similar? That makes it easier for the right people to notice them.

**Comment by ashokchandran** [comment]
> Can you prefix all your LTP patch PRs with "LTP: " or similar? That makes it easier for the right people to notice them.

done

**Review by shaikshavali1 — APPROVED** [review]
@ashokchandran, LGTM

File: tests/ltp/patches/fix_setreuid_setreuid07.patch [hunk 1]
```diff
@@ -65,19 +66,21 @@ index ff222cd00..57495d464 100644
...
++	if (TEST_RETURN != -1) {
++		tst_resm(TPASS,
++			"open call succeeded as expected in thread2");
++		close(tst_fd2);
++	} else {
++		tst_brkm(TBROK, cleanup,
++			"open failed unexpectedly in thread2");
++	}
++	if (tst_fd2 >= 0)
++		close(tst_fd2);
```
Comment (hukoyu) [inline tests/ltp/patches/fix_setreuid_setreuid07.patch:81]: Indentation is wrong

File: tests/ltp/patches/fix_setreuid_setreuid07.patch [hunk 2]
```diff
@@ -185,6 +188,8 @@ index ff222cd00..57495d464 100644
 +		        tst_resm(TPASS, "open call succeeded");
 +                        close(tst_fd);
```
Comment (hukoyu) [inline tests/ltp/patches/fix_setreuid_setreuid07.patch]: Indentation is wrong

File: tests/ltp/patches/fix_setreuid_setreuid07.patch [hunk 3]
```diff
@@ -65,19 +66,21 @@ index ff222cd00..57495d464 100644
 -static void do_master_child(void)
 +static void *do_sub_thread(void* arg)
 +{
-+        int tst_fd2;
++	int tst_fd2;
 +
 +        //Test to open the file in thread2 created by thread1
```
Comment (hukoyu) [inline tests/ltp/patches/fix_setreuid_setreuid07.patch:70]: Indentation is wrong

**Review by hukoyu — CHANGES_REQUESTED** [review]
@ashokchandran there are some indentation problems. Can you fix them?

**Review by hukoyu — APPROVED** [review]
LGTM

**Pull request merged**